infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/DefaultSnapshotSyncPlugin.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.infrastructure.logreplication.infrastructure.plugins;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Default Snapshot Sync Plugin
+ *
+ * This implementation returns immediately for start and end of a snapshot sync.
+ */
+@Slf4j
+public class DefaultSnapshotSyncPlugin implements ISnapshotSyncPlugin {
+
+    @Override
+    public void onSnapshotSyncStart() {
```

Comment:
Can this method take a CorfuRuntime as a parameter? Without that it's tricky to re-initialize a new runtime just for freezing checkpointing.